db/migrate/20151029052305_create_messages.rb

```diff
@@ -1,10 +1,11 @@
 class CreateMessages < ActiveRecord::Migration
```

Comment:
Hey @joryhatton, I've been blowing away my database any time I add properties to the model. Any tips or articles you'd suggest on how to update schema without going nuclear?